api/DataChannelMessage.h

```diff
@@ -132,6 +132,13 @@ inline void makeLastNEnd(utils::StringBuilder<1024>& outMessage)
 #endif
 }
 
+inline void makeMinUplinkBitrate(utils::StringBuilder<1024>& outMessage, const uint32_t maxBitrateKbps)
+{
+    outMessage.append("{\"type\":\"min-uplink-bitrate\", \"bitrateKbps\":\"");
```

Comment:
Try using R(""). It allows you to drop the \ and format the json in more readable form.

Reply:
It works :)